### DISCUSSION TITLE
Do not post support or feature requests here!

### DISCUSSION THREAD

**Discussion by RCheesley** [mautic/mautic#10425] [Announcements]
If you have a feature request, please submit it to the Ideas forum: https://forum.mautic.org/ideas

If you have a support request, please use the forums: https://forum.mautic.org/support

We do not currently want to use GitHub Discussions for either of these, and any submissions will be ignored and closed.

**Comment by dakur** [comment]
@RCheesley So what should it be used for? By the way, both links show 404.

**Comment by YosuCadilla** [comment]
I  don't think we should have yet another place where some stuff is allowed but some is not.
We already have "the forums" I don't see how having yet another place to discuss things is helpful.

**Comment by hannahbellelee** [comment]
I apologize in advance as I know you don't want discussions here, but several of the pages I visited were 404's and then I wound up deep in opencollective territory and that didn't seem right either. I was just wanting to ask about the releases and what the difference between the full 5.1.0 and the 5.1.0_update was. I wasnt sure if the update was like a hotfix to the main download or if it was an actual update from another version without having to install the full version... As you can tell, I'm kind of new here...lol...Ultimately im going to build it from the repository anyway, I just was curious. I can remove his post afterwards if you'd like. again, I'm sorry...
Hannah

**Comment by ramazansancar** [comment]
Could you add a login button with Github to the login screen of the forum system? @RCheesley

**Comment by panzer9** [comment]
Mautic 6 is just a terrible experience. I'm not a tech guy, but always managed to install Mautic 4 pretty easily. Now, with Mautic 6, Amazon SES e SNS support is terrible and importing is a nightmare, as apparently my workers are not working. 

Just deleted Mautic 6 installation and started from scratch with 4!
